### PR TITLE
Feature/add lefthook

### PR DESCRIPTION
**Pull request by marcel-strzalka**
This PR adds:

- Brakeman, Rubocop and Lefthook gems.
- A simple pre commit hook to run Brakeman and Rubocop.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -62,5 +62,10 @@ group :development do
 
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
+
+  # Static analyzers, linters and similar
+  gem "brakeman"
+  gem "rubocop"
+  gem "lefthook"
```
Comment (lilka) [inline Gemfile:69]: Add the required version like this , '~> 1.18'.

File: Gemfile [hunk 2]
```diff
@@ -62,5 +62,10 @@ group :development do
 
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
```
Comment (lilka) [inline Gemfile:64]: I think that you can delete it and other commended gems that you don't use ;)

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# The behavior of RuboCop can be controlled via the .rubocop.yml
+# configuration file. It makes it possible to enable/disable
+# certain cops (checks) and to alter their behavior if they accept
+# any parameters. The file can be placed either in your home
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://docs.rubocop.org/rubocop/configuration
```
Comment (lilka) [inline .rubocop.yml:10]: You can add here rubocop configuration and try it out ;)

**Review by lilka — APPROVED** [review]
I left some suggestions, but overall good job 👍

File: .rubocop.yml (hunk 3 above)
Comment (marcel-strzalka) [reply .rubocop.yml:10]: Gonna stick with the defaults for the moment, but thanks :D

**Pull request merged**